docs/getting_started_rules.md

```diff
@@ -0,0 +1,200 @@
+
+### Create new Rule
+Based on the use case and [rule category](./rules.md#categories), create a new Python file in the correct namespace of the [rules folder](/src/cfnlint/rules). Use the name of the Rule, so the filename of `MyNewRule` is `mynewrule.py`.
+
+Use the following skeleton code as a starting point of your new rule:
+
+```python
+"""
+  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this
+  software and associated documentation files (the "Software"), to deal in the Software
+  without restriction, including without limitation the rights to use, copy, modify,
+  merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+  permit persons to whom the Software is furnished to do so.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+  INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+  PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+from cfnlint import CloudFormationLintRule
+from cfnlint import RuleMatch
+
+
+class MyNewRule(CloudFormationLintRule):
+    """Rule description """
+    id = '' # New Rule ID
+    shortdesc = '' # A short description about the rule
+    description = '' # (Longer) description about the rule
+    source_url = '' # A url to the source of the rule, e.g. documentation, AWS Blog posts etc
+    tags = [] # A set of tags (strings) for searching
+
+    def match(self, cfn):
+        """Basic Rule Matching"""
+
+        matches = list()
+
+        # Your Rule code goes here
+
+        return matches
+```
+Fill in the Meta data (`id`, `shortdesc`, `description`, `source_url`, `tags`) of your new rule, these are required attributes needed for processing the rule.
+
+Since `cfn-lint` loads all rules from the `rules` folder as a plugin, the file and Class name must be the same.
+
+*At this point your new rule is already part of the linter's ruleset!*
+
+See the [code snippets](#code_snippets) section below for some examples of an implemention.
+
+### Test your new Rule
+You can test your new rule by just running `cfn-lint`, but preferably your new Rule is validated by unit tests. In this way the working is guaranteed for the future, and different use-cases can be validated automatically.
+
+#### Fixtures
+Fixture templates are created under the [test/templates/good](/test/templates/good) (for templates with valid CloudFormation code) or [test/templates/bad](/test/templates/bad) (for template with "invalid" CloudFormation Code) folders.  
```

Comment:
No need for quotes around "invalid" and you can remove capital letter for "Code"